lib/fbe/github_graph.rb

```diff
@@ -82,7 +82,7 @@ def resolved_conversations(owner, name, number)
         }
       GRAPHQL
     )
-    result.repository.pull_request.review_threads.to_h['nodes']
+    result&.repository&.pull_request&.review_threads&.to_h&.fetch('nodes', []) || []
```

Comment:
@Suban05 maybe use `dig` method for example:
```ruby
result&.to_h&.dig('repository', 'pull_request', 'review_threads', 'nodes) || []
```
this doesn't look very good: `&.fetch('nodes', []) || []`